packages/web-twig/src/Resources/components/CheckboxField/README.md

```diff
@@ -38,21 +38,24 @@ Without lexer:
-| `value`           | `string`                                                             | `null`  | no       | Input value                                                                                                                        |
+| Prop name           | Type                                                                 | Default | Required | Description                                                                                                                        |
+| ------------------- | -------------------------------------------------------------------- | ------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `class`             | `string`                                                             | `null`  | no       | Custom CSS class                                                                                                                   |
+| `id`                | `string`                                                             | `null`  | no       | Input and label identification                                                                                                     |
+| `isChecked`         | `bool`                                                               | `false` | no       | If true, input is checked                                                                                                          |
+| `isDisabled`        | `bool`                                                               | `false` | no       | If true, input is disabled                                                                                                         |
+| `isItem`            | `bool`                                                               | `false` | no       | To render in [Item][item] mode                                                                                                     |
+| `isLabelHidden`     | `bool`                                                               | `false` | no       | If true, label is hidden                                                                                                           |
+| `isRequired`        | `bool`                                                               | `false` | no       | If true, input is required                                                                                                         |
+| `label`             | `string`                                                             | —       | yes      | Label text                                                                                                                         |
+| `UNSAFE_label`      | `string`                                                             | —       | yes      | Unescaped label text                                                                                                               |
```

Comment:
I dont understand, how developer can use component by documentation. 
1) `Label` and `UNSAFE_label` are required
2) if developer not set label or unsefe_label - default(null) will be applied and component will be work (this is maybe for different discussion)

// after detail review, then the developer fills in both mandatory items, only unsafe variant will be show

Reply:
I have modified the docs to better describe this case. However, there is no way now how I can tell Twig that this prop is required. E.g. I think using the `default` in this case is the only option to not break things. We are working on a solution to tell the developer about the required prop.

Now the `label` is "required". And `UNSAFE_` variant has a higher priority and thus it will be used if both variants are set.